nemo_skills/evaluation/metrics/map_metrics.py

```diff
@@ -63,7 +63,7 @@
 }
 
 
-def get_metrics(metric_type: str):
+def get_metrics(metric_type: str, extra_arguments: dict = None):
```

Comment:
maybe just take it as **kwargs here? That would be consistent with other get_* functions we used throughout the codebase